apps/server/src/shared/repo/news/news-scope.ts

```diff
@@ -22,9 +22,13 @@ export class NewsScope extends Scope<News> {
 		return this;
 	}
 
-	byUnpublished(unpublished: boolean): NewsScope {
+	byUnpublished(unpublished: boolean, userId: EntityId | null = null): NewsScope {
```

Comment:
shouldnt userId should by a new scope and added in the repo?

Reply:
After consultations, stubs for unpublished and published new were made. Logic which stubs should be used was moved to news.repo.ts